### ISSUE TITLE
Internet Connection Status: error

### ISSUE THREAD

**Issue by dakipro** [breuerfelix/instapy-gui#112]
Hi, I tried running the container both on linux and windows, I always get 
ooooooooooooooooooooooooooo

INFO [2020-07-21 21:44:23] [instapy] Session started!
INFO [2020-07-21 21:44:23] [instapy] -- Connection Checklist [1/3] (Internet Connection Status)
WARNING [2020-07-21 21:44:51] [instapy] - Internet Connection Status: error
WARNING [2020-07-21 21:44:51] [instapy] - Internet Connection Status: error
CRITICAL [2020-07-21 21:44:51] [instapy] Unable to login to Instagram! You will find more information in the logs above.

I do have other containers that connect to the internet just fine. 
I've found this message to be sometimes related to the script itself, however it is a bit tricky to edit it if container shuts down immediately. Any thoughts on how to address this?

**Comment by breuerfelix** [comment]
this could be related to the check_browser function of instapy.

this is a knows issue there :)  
in the settings (where you put username and password) the last option is `want_check_browser`. set that to false and try again :)

**Comment by dakipro** [comment]
Thanks, I tried that but then I get other error, if I remember it was something like "waiting for login screen" which times out. 
I however tried to install plain instapy on windows, and got a bunch of errors that needed to be fixed by hacking some libraries, firefox gecko drivers and some more things I found on the internet. For now I am using the script on my main windows machine, but I was looking forward to the docker integration as it should be self contained

P.S. I see now that I posted this issue in the GUI repo, rather then in the docker. If it cannot be moved, I will create a new one.

**Comment by breuerfelix** [comment]
no problem, if you would have put that into the docker repo, i guess no one would have even noticed that :)

that seems to me that your docker container for real got no internet connection :)

could you try starting a normal bash container and curl google.com?

**Comment by dakipro** [comment]
Not in that container, as it stops before I can open console in it. But I do have a lot of containers on both machines running as expected, with internet connection and everything. 
Once I installed regular instapy, I got same problem but was able to fix it by replacing few lines in instapy library in python folder, few lines in selenium itself, and also a geckodriver version. I was hoping docker would be self contained regardless of firefox version or system I am running (the whole point of docker right), but for now I will most likely give it up as I do not have the time for it.
